packages/back-end/src/controllers/experiments.ts

```diff
@@ -1979,6 +1980,12 @@ export async function postSnapshotAnalysis(
 
   const metricMap = await getMetricMap(org.id);
 
+  if (snapshot.settings.coverage === undefined) {
+    const latestPhase = experiment.phases.length - 1;
+    snapshot.settings.coverage =
+      experiment.phases[phaseIndex ?? latestPhase].coverage;
+  }
```

Comment:
@jdorn if we stick with this, can I change this to update the snapshot itself with the coverage so this acts as a JIT migration?

Reply:
yeah, that's fine

Reply:
Done, I update the whole settings object. Given I just fetched the snapshot this shouldn't be too bad!